enterprise/backend/test/metabase_enterprise/serialization/api_test.clj

```diff
@@ -118,33 +124,35 @@
                            Dashboard  _dash {:collection_id (:id coll)}
                            Card       card  {:collection_id (:id coll)}]
 
-              (let [res    (mt/user-http-request :crowberto :post 200 "ee/serialization/export"
-                                                 :collection (:id coll) :data_model false :settings false)
-                    files* (atom [])
-                    ;; to avoid input stream closure
-                    ba     (#'api.serialization/ba-copy (io/input-stream res))]
-                (with-open [tar (open-tar ba)]
-                  (doseq [^TarArchiveEntry e (u.compress/entries tar)]
-                    (when (.isFile e)
-                      (swap! files* conj (.getName e)))
-                    (condp re-find (.getName e)
-                      #"/export.log$" (testing "Three lines in a log for data files"
-                                        (is (= 3 (count (line-seq (io/reader tar))))))
-                      nil)))
-
+              (let [res (-> (mt/user-http-request :crowberto :post 200 "ee/serialization/export"
+                                                  :collection (:id coll) :data_model false :settings false)
+                            io/input-stream)
+                    ;; we're going to re-use it for import, so a copy is necessary
+                    ba  (#'api.serialization/ba-copy res)]
                 (testing "We get only our data and a log file in an archive"
-                  (is (= 4 (count @files*))))
+                  (is (= 4
+                         (with-open [tar (open-tar ba)]
+                           (count
+                            (for [^TarArchiveEntry e (u.compress/entries tar)
+                                  :when              (.isFile e)]
+                              (do
+                                (condp re-find (.getName e)
+                                  #"/export.log$" (testing "Three lines in a log for data files"
+                                                    (is (= (+ #_extract 3 #_store 3)
+                                                           (count (line-seq (io/reader tar))))))
+                                  nil)
+                                (.getName e))))))))
 
                 (testing "Snowplow export event was sent"
-                  (is (=? {"event"           "serialization"
```

Comment:
I don't think you're allowed to remove keys from snowplow events, you'd need to deprecate the schema to do this, which doesn't seem worth it.

Reply:
oh; hmmm, how do I learn about this for sure...

Reply:
I've put them back in :)